modules/php/Models/Scoresheets/Scoresheet2.php

```diff
@@ -107,6 +111,23 @@ public static function phase5Check(): void
     Globals::setCircledMultipliers([]);
   }
 
+  public function isWriteXOptional(): bool
+  {
+    return true;
+  }
+
+  public function getScribbleReactions($scribble): array
+  {
+    if (Players::getCurrent()->scoresheet()->hasScribbledSlot($scribble->getSlot(), NUMBER_X)) {
```

Comment:
Why not directly $scribble->getType() == NUMBER_X ?

Reply:
Somehow I haven't found this, thanks :)